calliope/backend/run.py

```diff
@@ -525,6 +541,7 @@ def run_operate(model_data, timings, backend, build_only):
                 _results = backend.get_result_array(backend_model, model_data)
             else:
                 _results = xr.Dataset()
+                continue
```

Comment:
is this a good idea? it wasn't there before and if we need it now, maybe the logic should be rewritten so it's clearer what's happening?